cmake/FindJSON.cmake

```diff
@@ -0,0 +1,21 @@
+include(FetchContent)
+
+message(STATUS "Fetching json...")
+
+FetchContent_Declare(
+	json
+	GIT_REPOSITORY https://github.com/nlohmann/json
+	GIT_TAG v3.11.3
+)
+
+FetchContent_GetProperties(json)
+if(NOT json_POPULATED)
+	set(JSON_BuildTests OFF CACHE INTERNAL "")
```

Comment:
```suggestion
```
looked into this some, it's not needed here, their cmake handles it if it's being used externally and is not the main project

Reply:
https://github.com/nlohmann/json/blob/develop/CMakeLists.txt#L35-L39